iceberg/iceberg-rest-server/src/main/java/org/apache/gravitino/iceberg/service/IcebergRestUtils.java

```diff
@@ -122,6 +126,19 @@ public static NameIdentifier getGravitinoNameIdentifier(
     return NameIdentifier.of(catalogNS.toArray(String[]::new));
   }
 
+  public static Map<String, String> getHttpHeaders(HttpServletRequest httpServletRequest) {
+    Map<String, String> headers = new HashMap<>();
+    Enumeration<String> headerNames = httpServletRequest.getHeaderNames();
+    while (headerNames.hasMoreElements()) {
+      String headerName = headerNames.nextElement();
+      String headerValue = httpServletRequest.getHeader(headerName);
+      if (headerValue != null) {
```

Comment:
Curious is there a chance that headerValue can be null here?

Reply:
According to the API, the headerValue will be null if `headerName` is not in HTTP request headers. In theory the headerValue will not be null in current implement, but it's safe to add the check here.

Reply:
Thanks!